packages/react-img-viewer/.gitignore

```diff
@@ -0,0 +1,8 @@
+*.log
+.DS_Store
+node_modules
+.cache
+dist
+src/stories
```

Comment:
Q: Why have we have added src/stories in .gitignore.

Reply:
I added stories in .gitignore in start then i forgot to remove after work was complete.
thanks for notifying me, i will remove it from the .gitignore